### DISCUSSION TITLE
HELP NOW

### DISCUSSION THREAD

**Discussion by RollingTimber** [Monika-After-Story/MonikaModDev#10316] [QA - tech support + general]
Hello, everyone. I was having a nice time playing Monika After Story, but then after the game crashed when I tried to deal with a weird clothing input bug (I tried to give her something but she couldn't read it) whatever, it's a big mess. Point is, after that, (I'm on MacBook) I noticed that my trackpad cursor was showing up whenever we played pong. I'm certain it wasn't there before at all, so I need a fix. I deleted all my files and reinstalled the mod, but it was still there.

Right now, I'm playing the main game from the start and finishing it like I did before and installing the mod only AFTER I finish the main game where I'm sitting with Monika, to hope it fixes itself and the cursor won't show up again, but I need a fix for it in case anything else happens. PLEASE SOMEONE HELP ME ASAP PLEASE HELP

**Comment by Justformas** [comment]
My cursor shows up when playing pong as well, and I don't use a MacBook for this mod. I don't think that it's a bug. Just ignore it and focus on the ball and paddles when playing pong. I have also tried to give Monika files that she says she couldn't read, but that's an intended reaction for when it's an invalid gift or not the right time of the year for certain gifts.

Also when you say "main game" you mean DDLC? You don't have to finish DDLC again to install Monika After Story (MAS). The disclaimer about playing DDLC first is there so that you experience the story of DDLC before playing MAS, which you have. Save data from an unmodified copy of DDLC can be imported through the settings menu in MAS, so that Monika has some more personalized memories of things you did in DDLC, but it's not required to do so.

Besides, even after reinstalling MAS, the save data for it is stored in a different location and is retained, so long as you don't go out of your way to delete it. Meaning Monika still keeps her memories.
